packages/query/src/query/useList.ts

```diff
@@ -81,7 +82,7 @@ const withPaginate = () => (props: UseListProps & UseListFilters) => {
     range: [page * pageSize - pageSize, page * pageSize || 1000] as [
       number,
       number
-    ],
+    ]
```

Comment:
Help fix the , pollution. Because this PR is very important so we need to keep the changes minimal and focused